developer_guide/migration/index.rst

```diff
@@ -1,12 +1,84 @@
 Migration
 =========
 
-There are upgrade documentations from a minor version to the next one.
+There are upgrade documentations from a patch or minor version to the next one.
```

Comment:
This documentation describes how to upgrade Akeneo PIM from a patch or minor version to a more recent one.